requirements.txt

```diff
@@ -1,31 +1,46 @@
 alembic==0.9.6
+astroid==1.5.3
+backports.functools-lru-cache==1.4
 bcrypt==3.1.4
 cffi==1.11.2
 click==6.7
+colorama==0.3.9
+configparser==3.5.0
 Django==1.11.7
+django-bootstrap3==9.1.0
 dominate==2.3.1
+enum34==1.1.6
 Flask==0.12.2
 Flask-Bcrypt==0.7.1
 Flask-Bootstrap==3.3.7.1
 Flask-Migrate==2.1.1
+Flask-Script==2.0.6
 Flask-SQLAlchemy==2.3.2
 Flask-WTF==0.14.2
 inflect==0.2.5
+isort==4.2.15
 itsdangerous==0.24
 Jinja2==2.9.6
+lazy-object-proxy==1.3.1
 Mako==1.0.7
 MarkupSafe==1.0
+mccabe==0.6.1
 olefile==0.44
+pep8==1.7.1
 Pillow==4.3.0
 psycopg2==2.7.3.2
+pycodestyle==2.3.1
 pycparser==2.18
+pylint==1.7.4
 python-dateutil==2.6.1
 python-editor==1.0.3
 pytz==2017.3
+singledispatch==3.4.0.3
 six==1.11.0
 sqlacodegen==1.1.6
 SQLAlchemy==1.1.14
 SQLAlchemy-Utils==0.32.19
 visitor==0.1.3
 Werkzeug==0.12.2
+wrapt==1.10.11
 WTForms==2.1
```

Comment:
you added a lot of  packages. They exactly needded?

Reply:
main part of this packages are nedded for pylint